.byebug_history

```diff
@@ -1,4 +1,22 @@
 exit
+Order.all
```

Comment:
hmm, why is byebug history suddenly showing up? shouldn't this be in .gitignore?